src/d2l-iframe-wrapper-for-react.js

```diff
@@ -1,6 +1,6 @@
 import { LitElement, html, css } from 'lit-element';
 import * as ResizeCallbackMaker from './ResizeCallbackMaker.js';
-import { styleMap } from 'lit-html/directives/style-map';
+import { styleMap } from 'lit-html/directives/style-map.js';
```

Comment:
This should be `import { styleMap } from 'lit/directives/style-map.js';`. And any other `import` from `lit-element` or `lit-html` should be changed to `lit`.